internal/cmd/group_tokens.go

```diff
@@ -131,13 +131,26 @@ var groupCreateTokenCmd = &cobra.Command{
 	},
 }
 
-func validateDBNames(dbNames []string) error {
-	databases := map[string]string{}
-	for _, db := range getDatabasesCache() {
-		databases[db.Name] = db.ID
+func validateDBNames(client *turso.Client, dbNames []string) error {
```

Comment:
⛏️: you're doing the same logic twice here, I suggest creating a function that received the database map and db names and checks if any is missing. Then you call it with cached values and call it again with fresh if the first one failed.